configs/body3d/videopose/h36m/videopose_h36m_27frames_fullconv_semi-supervised.py

```diff
@@ -68,11 +68,12 @@
         type='TemporalRegressionHead',
         in_channels=1024,
         num_joints=1,
-        loss_keypoint=dict(type='MPJPELoss', use_target_weight=True)),
+        loss_keypoint=dict(type='MPJPELoss', use_target_weight=True),
+        is_trajectory=True),
     loss_semi=dict(
         type='SemiSupervisionLoss',
-        joint_parents=[0, 0, 1, 2, 0, 4, 5, 0, 7, 8, 9, 8, 11, 12, 8, 14, 15]),
-    warmup_epochs=1,
+        joint_parents=[0, 0, 1, 2, 0, 4, 5, 0, 7, 8, 9, 8, 11, 12, 8, 14, 15],
+        warmup_iterations=12805),  # num_GPUs = 8
```

Comment:
**12805** a magic number?

Reply:
This is a workaround. Because currently, we cannot access epoch number in loss function. So we use iteration number instead.

Reply:
Can this be implemented by hooks?

Reply:
@innerlee @jin-s13 Yes, this can be done by hooks. We plan to add a LossGateHook or something like that, to control the loss backpropagation by conditions like epoch/iteration number. It would be implemented in a future PR and need some design. This is a workaround for this algorithm.

@cherryjm Could you please change it to an equation like warmup_iterations=dataset_size/batch_size/GPU_num, and explain each number in the comment? That would be more clear.

Reply:
OK. Will modify it.